config/samples/csi_v1_hostdefiner.yaml

```
@@ -0,0 +1,36 @@
+apiVersion: csi.ibm.com/v1
+kind: HostDefiner
+metadata:
+  name: host-definer
+  namespace: default
+  labels:
+    app.kubernetes.io/name: host-definer
+    app.kubernetes.io/instance: ibm-block-csi
+    app.kubernetes.io/managed-by: ibm-block-csi-operator
+    release: v1.10.0
+spec:
+  hostDefiner:
+    #prefix:
+    #connectivityType:
+    #allowDelete:         #Default true
```

```suggestion
    #allowDelete: true    # Optional. Values true/false. The default is true.
```
same as in the [driver example](https://github.com/IBM/ibm-block-csi-driver/blob/v1.9.0/deploy/kubernetes/examples/demo-storageclass.yaml#L12)s

---

but it's bool, it's known that his only options are true or false

---

is doesn't say it's a "bool", as far a the user knows, it could be a string

---

done